examples/multi_inspiral/clean.sh

```diff
@@ -1 +1 @@
-rm -f bank_veto_bank.xml *.hdf *.gwf
```

Comment:
Are the frame files deleted in another way now?

Reply:
Deleting them is not necessary because they are not even saved in the current directory. They are cached under `~/.astropy/cache/`.